### PR TITLE
Add email_subject_prefix directive

### PR DESCRIPTION
**Pull request by Gr33ny**
https://github.com/golemfactory/concent-deployment/issues/39

### CODE REVIEW

File: kubernetes/config-maps/concent-api/local_settings.py.j2 [hunk 1]
```diff
@@ -30,6 +30,7 @@ ADMINS = [
 EMAIL_HOST          = "{{ email.host }}"
 EMAIL_PORT          = {{ email.port }}
 EMAIL_HOST_USER     = "{{ email.user }}"
+EMAIL_SUBJECT_PREFIX = "[concent-api-{{ email.subject }}] "
```
Comment (cameel) [inline kubernetes/config-maps/concent-api/local_settings.py.j2]: Let's do it a bit differently, without adding a new variable. Please remove the `email.subject`. Then change the value to: `"[{{ gke.cluster }}:concent-api] "`.

File: kubernetes/config-maps/gatekeeper/local_settings.py.j2 [hunk 2]
```diff
@@ -27,6 +27,7 @@ ADMINS = [
 EMAIL_HOST          = "{{ email.host }}"
 EMAIL_PORT          = {{ email.port }}
 EMAIL_HOST_USER     = "{{ email.user }}"
+EMAIL_SUBJECT_PREFIX = "[gatekeeper-{{ email.subject }}] "
```
Comment (cameel) [inline kubernetes/config-maps/gatekeeper/local_settings.py.j2]: Change the value to: `"[{{ gke.cluster }}:gatekeeper] "`.

File: kubernetes/config-maps/concent-api/local_settings.py.j2 [hunk 3]
```diff
@@ -30,6 +30,7 @@ ADMINS = [
 EMAIL_HOST          = "{{ email.host }}"
 EMAIL_PORT          = {{ email.port }}
 EMAIL_HOST_USER     = "{{ email.user }}"
+EMAIL_SUBJECT_PREFIX = "[{{ gke.cluster }}-concent-api] "
```
Comment (cameel) [inline kubernetes/config-maps/concent-api/local_settings.py.j2]: This is not what I requested in https://github.com/golemfactory/concent-deployment/pull/41#discussion_r163215396

Comment (Gr33ny) [reply kubernetes/config-maps/concent-api/local_settings.py.j2]: Ok, I see. Replace "-"=> ":"

**Pull request merged**